### ISSUE TITLE
[CommandBar] Accessibility: While swiping, talkback focus is moving out of the expanded menu options & reading background controls

### ISSUE THREAD

**Issue by lherrle** [OfficeDev/office-ui-fabric-react#6120]
### Please provide a reproduction of the bug in a codepen:
 
Repro is attached to this bug:
https://dynamicscrm.visualstudio.com/OneCRM/_workitems/edit/1176139

### Bug Report 

- __Package version(s)__: 5.112.0
- __Browser and OS versions__: Android Phone 8.0 with talkback
 
#### Priorities and help requested:
 
Are you willing to submit a PR to fix? No
 
Requested priority: Normal
 
Products/sites affected: Dynamics 365 Project Service
 
#### Describe the issue:

When using Talkback to swipe through the overflow buttons, the focus leaves the menu without closing the popup. This is different than tabbing behavior with the keyboard.

User Impact
Screen Reader users will not be able to close the expanded popup menu logically, and they will find it difficult to understand the sequence of controls on the page.

MAS Reference
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={1ee227a7-e3c5-47b8-ab18-48ca8595077b}
 
#### Actual behavior:
 
Talkback focus is moving out of the expanded menu items (three dots control) and hence read background controls after the last menu item.
 
#### Expected behavior:
 
Talkback focus should not move out of the popup menu. After last menu option, an option should be provided to collapse the expanded popup (for next swipe).
